tests/python/pants_test/reporting/test_reporting_integration.py

```diff
@@ -257,6 +257,31 @@ def test_zipkin_reporter_for_v2_engine(self):
         v2_span_name_part, trace
         ))
 
+  def test_zipkin_reports_for_pure_v2_goals(self):
+    ZipkinHandler = zipkin_handler()
+    with http_server(ZipkinHandler) as port:
+      endpoint = "http://localhost:{}".format(port)
+      command = [
+        '--v1',
```

Comment:
I think the flags should be `--no-v1 --v2`.

Reply:
+1 - though in this case the two are actually equivalent.

Goals may be implemented in only v1, only v2, or both.

List is only implemented in v2 (there are no v1 tasks to run).

But hypothetically the `test` goal may have python test support implemented in v2, but java test support implemented in v1; if you ran `--v1 --v2` test you would run both, if you ran `--no-v1 --v2` test you would only run python tests but not java tests.

I can't imagine `list` also having any v1 tasks, so as I say these `--v1` and `--no-v1` are equivalent if the only goal is `list`, but `--no-v1` better expresses the intent of the test :)